WordPress/Classes/Utility/WPError.m

```diff
@@ -32,12 +32,48 @@ + (void)showNetworkingAlertWithError:(NSError *)error
 }
 
 + (void)showNetworkingAlertWithError:(NSError *)error title:(NSString *)title
+{
+    if ([self showWPComSigninIfErrorIsInvalidAuth:error]) {
+        return;
+    }
+    
+    NSDictionary *titleAndMessage = [self titleAndMessageFromNetworkingError:error desiredTitle:title];
+    
+    [self showAlertWithTitle:titleAndMessage[@"title"]
+                     message:titleAndMessage[@"message"]
+           withSupportButton:YES
+              okPressedBlock:nil];
+}
+
++ (void)showXMLRPCErrorAlert:(NSError *)error
```

Comment:
This is misleading. I never changed this method.

Reply:
Did you move it?

Reply:
I moved most of the contents of the other method above: `showNetworkingAlertWithError`.